src/main/java/com/flowingcode/vaadin/addons/gridexporter/GridExporter.java

```diff
@@ -168,21 +179,24 @@ public static <T> GridExporter<T> createFor(
                   docLink.setHref(
                       exporter.getDocxStreamResource(docxCustomTemplate).forComponent(docLink));
                   docLink.getElement().setAttribute("download", true);
+                  exporter.applyTooltip(docLink, exporter.docxIconTooltip);
                   footerToolbar
                       .add(new FooterToolbarItem(docLink, FooterToolbarItemPosition.EXPORT_BUTTON));
                 }
                 if (exporter.isPdfExportEnabled()) {
-                  Anchor docLink = new Anchor("", FontAwesome.Regular.FILE_PDF.create());
-                  docLink.setHref(
-                      exporter.getPdfStreamResource(docxCustomTemplate).forComponent(docLink));
-                  docLink.getElement().setAttribute("download", true);
+                  Anchor pdfLink = new Anchor("", FontAwesome.Regular.FILE_PDF.create());
+                  pdfLink.setHref(
+                      exporter.getPdfStreamResource(null).forComponent(pdfLink));
```

Comment:
`exporter.getPdfStreamResource(docxCustomTemplate)` was replaced by `exporter.getPdfStreamResource(null)`. This is likely a mistake or an unrelated fix.